packages/flutter_tools/test/general.shard/build_system/targets/assets_test.dart

```diff
@@ -65,7 +67,7 @@ flutter:
 
     // See https://github.com/flutter/flutter/issues/35293
     expect(fs.file(fs.path.join(environment.buildDir.path, 'flutter_assets', 'assets/foo/bar.png')).existsSync(), false);
-  }));
+  }), skip: Platform.isWindows); // See https://github.com/google/file.dart/issues/131
```

Comment:
Does this imply that stale files will be left in the build directory on Windows? Could this result in an incorrect build?

Reply:
I could not reproduce this on a real filesystem, which leads me to believe it is only an issue with the memory filesystem.